pkg/cmd/multi-operator-manager/test/apply-configuration/apply_configuration_command.go

```diff
@@ -127,3 +135,19 @@ func (f *TestApplyConfigurationFlags) ToOptions(ctx context.Context) (*testapply
 		Streams:         f.Streams,
 	}, nil
 }
+
+func validateTest(path *field.Path, testOptions *testapplyconfiguration.TestOptions) []error {
+	errs := []error{}
+
+	if len(testOptions.Description.TestName) == 0 {
+		errs = append(errs, field.Required(path.Child("testName"), "must be present"))
+	}
+	if len(testOptions.Description.Description) == 0 {
+		errs = append(errs, field.Required(path.Child("description"), "must be present"))
+	}
+	if !testapplyconfiguration.AllTestTypes.Has(testOptions.Description.TestType) {
```

Comment:
is this test (apply_configuration) allowed to define a different type than `ApplyConfiguration` ?

Reply:
>is this test (apply_configuration) allowed to define a different type than ApplyConfiguration ?

I suspect we eventually build one for apply-configuration-live.  requiring differnet directories?  maybe.  But why allow an accident?